gnat2goto/driver/gnat2goto.ads

```diff
@@ -0,0 +1,2 @@
+package GNAT2GOTO is
```

Comment:
This is only here because I wanted to call it `GNAT2GOTO.Options` rather than just `Options`...